.circleci/config.yml

```diff
@@ -0,0 +1,58 @@
+version: 2.1
+
+commands:
+  prepare:
+    parameters:
+      symfony_version:
+        type: string
+    steps:
+      - checkout
+      - run: sudo composer self-update
+      - run: composer require symfony/symfony:<< parameters.symfony_version >> --no-update
+      - restore_cache:
+          keys:
+            - composer-v1-{{ checksum "composer.json" }}
+            - composer-v1-
+      - run: composer install -n --prefer-dist
+      - run: composer update --prefer-dist --no-interaction
+      - persist_to_workspace:
+          root: .
+          paths:
+            - composer.*
+      - save_cache:
+          key: composer-v1-{{ checksum "composer.json" }}
+          paths:
+            - vendor
+
+jobs: 
+  php_7: 
+    docker: 
+      - image: circleci/php:7.1
+    steps:
+      - prepare:
+          symfony_version: 3.3.*
```

Comment:
This might be the problem ... I'm guessing this came from the previous CI config? Not sure why this is being frozen at 3.